### PR TITLE
Adds fixes related to gameplay and bugs

### PR DESCRIPTION
**Pull request by nk3nny**
### GAMEPLAY
Added chance for crouched stance when assaulting (for variety)
Fixed prevents unit in vehicles from stopping and turning to stealth mode 
Fixed refeted cfgFSM settings to 26,8 to increase unit cohesion 
Fixed more time for check body event 
Fixed use 2D position for speed changes in CQC. Makes unit speed changes fewer
Fixed added more 'SuppressFor' time in fnc_suppress
Fixed store stance in FSM to prevent reset
Fixed improved panic state by inserting civilian animation switchmove

### CODE REVIEW

File: addons/danger/functions/debug/fnc_debugDraw.sqf [hunk 1]
```diff
@@ -184,15 +184,15 @@ private _fnc_DrawRect = {
...
+            "Behaviour: ", behaviour _unit, "<br/>",
+            "Current Task: ", _unit getVariable [QGVAR(currentTask), "None"], "<br/>"
         ];
         if (_unit == leader _unit) then {
             private _dangeModeData = (group _unit) getVariable [QGVAR(dangerMode), [[], [], true, time]];
             private _queue = (_dangeModeData select 0) apply { _x call _fnc_dangerModeTypes };
             _textData append [
                 "Danger Mode Queue: ", [_queue, "None"] select (_queue isEqualTo []), "<br/>",
-                "Danger Mode Timeout: ", round ((_dangeModeData select 3) *100)/100, "<br/>"
+                "Danger Mode Timeout: ", round ( (_dangeModeData select 3) - time ), "s <br/>"
```
Comment (jokoho48) [inline addons/danger/functions/debug/fnc_debugDraw.sqf]:
```suggestion
               format [ "Danger Mode Timeout: %1s <br/>", round ( (_dangeModeData select 3) - time )]
```

File: addons/danger/functions/fnc_checkBody.sqf [hunk 2]
```diff
@@ -65,7 +65,7 @@ if !(_body isEqualTo []) exitWith {
                 _unit doFollow leader group _unit;
```
Comment (jokoho48) [inline addons/danger/functions/fnc_checkBody.sqf]: when the Unit is alive and time is up but the distance is still bigger than 0.8 he continues to walk towards the target but does not do anything after that.
we maybe should move the doFollow always if the unit is alive

Comment (nk3nny) [reply addons/danger/functions/fnc_checkBody.sqf]: Hmm. Not a bad idea. Rarely comes up as units which haven't reached the body yet have probably been given new orders. 

Perhaps rewrite a bit. With the waitUntilAndExecute syntax the timeout is a bit redundant in any case. https://cbateam.github.io/CBA_A3/docs/files/common/fnc_waitUntilAndExecute-sqf.html

File: addons/danger/functions/fnc_assaultSpeed.sqf [hunk 3]
```diff
@@ -19,11 +19,11 @@
 params ["_unit", ["_target",ObjNull]];
 
 // distance
-private _distance = _unit distance _target;
+private _distance = _unit distance2d _target;
 
 // speed
 if (behaviour _unit isEqualTo "STEALTH") exitWith {1};
-if (speedMode _unit isEqualTo "FULL") exitWith {[24, 3] select (_distance < (8 + random 4))};
+if (speedMode _unit isEqualTo "FULL") exitWith {[24, 3] select (_distance2d < (8 + random 4))};
```
Comment (jokoho48) [inline addons/danger/functions/fnc_assaultSpeed.sqf]: this variable does not exist...

Comment (nk3nny) [reply addons/danger/functions/fnc_assaultSpeed.sqf]: seen

File: addons/danger/functions/fnc_isAlive.sqf [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+ * Return Value:
+ * boolean
+ *
+ * Example:
+ * bob call lambs_danger_fnc_isAlive;
+ *
+ * Public: No
+*/
+alive _this
+|| {!(_this getVariable ["ACE_isUnconscious", false])}
```
Comment (diwako) [inline addons/danger/functions/fnc_isAlive.sqf]: This one is not needed anymore, unless you want to support ACE versions before the Medical rewrite, or people running old ace medical.

Comment (jokoho48) [reply addons/danger/functions/fnc_isAlive.sqf]: I think we should support it because there are still a few communities running ace 3.12.x

**Pull request merged**